containerd-shim-v2/service.go

```diff
@@ -298,12 +299,19 @@ func (s *service) Start(ctx context.Context, r *taskAPI.StartRequest) (*taskAPI.
 		}
 
 		return &taskAPI.StartResponse{
-			Pid: PID,
+			Pid: s.pid,
```

Comment:
Nit: This is supposed to be part of the "start" implementation commit.

Reply:
Done